jwst/persistence/persistence_step.py

```diff
@@ -20,23 +18,31 @@ class PersistenceStep(Step):
         save_persistence = boolean(default=False) # Save subtracted persistence to an output file with suffix '_output_pers'
         save_trapsfilled = boolean(default=True) # Save updated trapsfilled file with suffix '_trapsfilled'
         modify_input = boolean(default=False)
-    """ # noqa: E501
+    """  # noqa: E501
 
     reference_file_types = ["trapdensity", "trappars", "persat"]
 
     def process(self, step_input):
-
+        """
+        Execute the persistence correction step.
+
+        Parameters
+        ----------
+        step_input : DataModel
```

Comment:
Can the step be run standalone, and if so, can this also be `str`?

Reply:
Yes, can also be a str